tuf/repository_tool.py

```diff
@@ -1758,32 +1758,33 @@ def target_files(self):
 
 
 
-  def add_restricted_paths(self, restricted_paths, child_rolename):
+  def add_paths(self, paths, child_rolename):
     """
     <Purpose>
-      Add 'restricted_paths' to the restricted paths of 'child_rolename'.
-      The updater client verifies the target paths specified by child roles, and
-      searches for targets by visiting these restricted paths.  A child role may
-      only provide targets specifically listed in the delegations field of the
-      parent, or a target that matches a restricted path.
+      Add 'paths' to the delegated paths for 'child_rolename'.  'paths' can be
+      a list of either file paths or glob patterns.  The updater client
+      verifies the target paths specified by child roles, and searches for
+      targets by visiting these delegated paths.  A child role may only provide
+      targets specifically listed in the delegations field of the parent role,
+      or a target that matches a delegated path.
 
       >>>
       >>>
       >>>
 
     <Arguments>
-      restricted_paths:
-        A list of paths that 'child_rolename' should be restricted to.
+      paths:
+        A list of glob patterns, or file paths, that 'child_rolename' is
+        trusted to provide.
 
       child_rolename:
-        The child delegation that requires an update to its restricted paths,
-        as listed in the parent role's delegations (e.g., 'Django' in
-        'unclaimed').
+        The child delegation that requires an update to its delegated or
+        trusted paths, as listed in the parent role's delegations (e.g.,
+        'Django' in 'unclaimed').
 
     <Exceptions>
-      securesystemslib.exceptions.Error, if a restricted path in
-      'restricted_paths' is not a string path, or if 'child_rolename' has not
-      been delegated yet.
+      securesystemslib.exceptions.Error, if a delegated path in 'paths' is not
+      a string path, or if 'child_rolename' has not been delegated yet.
```

Comment:
The first condition actually raises a `FormatError`, right? Also, I'd add *"...or not a glob pattern."*. But in reality the `PATHS_SCHEMA` only checks for a list of `AnyStrings` (could be empty too).